cmd/nephe-controller/options.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"github.com/spf13/pflag"
+	"gopkg.in/yaml.v2"
+	"os"
+)
+
+const (
+	namePrefix = "nephe-"
+)
+
+type Options struct {
+	// The path of configuration file.
+	configFile string
+	// The configuration object
+	config *ControllerConfig
+}
+
+func newOptions() *Options {
+	return &Options{
+		config: &ControllerConfig{},
```

Comment:
Maybe we can set the default values here then return. This saves checks for setting default later in `complete()` and imo its more readable.